sap-spring-properties/src/main/java/com/sap/cloud/environment/servicebinding/SapSpringPropertiesServiceBindingAccessor.java

```diff
@@ -0,0 +1,97 @@
+package com.sap.cloud.environment.servicebinding;
+
+import com.sap.cloud.environment.servicebinding.api.DefaultServiceBinding;
+import com.sap.cloud.environment.servicebinding.api.ServiceBinding;
+import com.sap.cloud.environment.servicebinding.api.ServiceBindingAccessor;
+import com.sap.cloud.environment.servicebinding.api.exception.ServiceBindingAccessException;
+import com.sap.cloud.environment.servicebinding.environment.SapServiceBindingsPropertiesAccessor;
+import com.sap.cloud.environment.servicebinding.SapServiceBindingsProperties.ServiceBindingProperties;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+/**
+ * A {@link ServiceBindingAccessor} that is able to load {@link SapServiceBindingsProperties}s from Spring's application
+ * properties.
+ */
+public class SapSpringPropertiesServiceBindingAccessor implements ServiceBindingAccessor
+{
+    @Nonnull
+    private static final Logger logger = LoggerFactory.getLogger(SapSpringPropertiesServiceBindingAccessor.class);
+
+    @Nonnull
+    @Override
+    public List<ServiceBinding> getServiceBindings()
+        throws ServiceBindingAccessException
+    {
+        List<ServiceBinding> serviceBindings = fetchServiceBindings();
+
+        serviceBindings
+            .forEach(binding -> logger.debug("Service binding {} found in properties.", binding.getServiceName().get()) //NOSONAR
+            );
+
+        return serviceBindings;
+    }
+
+    @Nonnull
+    private List<ServiceBinding> fetchServiceBindings()
+    {
+        Map<String, ServiceBindingProperties> serviceBindingsProperties =
+            SapServiceBindingsPropertiesAccessor.getServiceBindingsProperties();
+
+        if( serviceBindingsProperties.isEmpty() ) {
+            logger.info("No service bindings found in properties.");
```

Comment:
Seems to be a similar log line as in the environment post processor if the result is not bound.